docs/manual/java/guide/advanced/IntegratingNonLagom.md

```diff
@@ -26,12 +26,18 @@ Of course, you will also need to add a dependency to the API project that you ha
 
 The Lagom integration client provides [`LagomClientFactory`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html) creating Lagom client services.  This factory creates and manages thread pools and connection pools, so it's important to manage its lifecycle correctly in your application, ensuring that you only create one instance of it, and to shut it down when you're finished with it.
 
-The factory can be instantiated by invoking the static [`create`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html#create-java.lang.String-java.lang.ClassLoader-) method, for example:
+The factory can be instantiated by invoking the static [`create(String, ClassLoader)`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html#create-java.lang.String-java.lang.ClassLoader-) or [`create(String, ClassLoader, ActorSystem, Materializer)`](api/index.html?com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.html#create-java.lang.String-java.lang.ClassLoader-akka.actor.ActorSystem-akka.stream.Materializer-) methods:
```

Comment:
note to self: this is ugly when printed on the site is generated. I will re-work that part.